src/providers/data_providers/businessImageProviderFactory.ts

```diff
@@ -0,0 +1,34 @@
+import { BusinessImageProvider } from './businessImageProvider';
+import { ImageProviderName } from '../../models/data_models/image';
+import { MockBusinessImageProvider } from './mockBusinessImageProvider';
+import { YelpBusinessImageProvider } from './yelpBusinessImageProvider';
+import { Image } from '../../models/data_models/image';
+import { BusinessInfoInput } from '../../models/data_models/types';
+
+export class BusinessImageProviderFactory {
```

Comment:
This isn't really a factory anymore if the getProvider functions are private; it's closer to an aggregator